src/lightning_app/cli/commands/connection.py

```diff
@@ -284,3 +311,27 @@ def _clean_lightning_connection():
             connection = os.path.join(_LIGHTNING_CONNECTION, str(ppid))
             if os.path.exists(connection):
                 shutil.rmtree(connection)
+
+
+def _scan_lightning_connections(app_name_or_id):
+    if not os.path.exists(_LIGHTNING_CONNECTION):
+        return
+
+    for ppid in os.listdir(_LIGHTNING_CONNECTION):
+        try:
+            psutil.Process(int(ppid))
+        except (psutil.NoSuchProcess, ValueError):
+            continue
```

Comment:
do we need these lines ?

Reply:
It’s convenient to avoid peeking into files and checking for the app name/id for all connections.
If the pid doesn’t correspond to any active terminal process, we just skip the rest.